packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceCreateManifest.test.ts

```diff
@@ -184,6 +186,46 @@ describe('Force Create Manifest', () => {
       expect(openTextDocumentSpy.calledOnce).to.equal(true);
     });
 
+    describe('Verify manifest version.', () => {
+      const fakeVersion = '54.0';
+      let getValueStub: sinon.SinonStub;
+
+      beforeEach(() => {
+        getValueStub = env
+          .stub(SfdxProjectConfig, 'getValue')
+          .resolves(fakeVersion as unknown);
+      });
+
```

Comment:
I think you should add afterEach block here. This would ensure that environment is restored after each test. Something like this : 
```
afterEach(() => {
      env.restore();
    });
```

Reply:
Due to the describe block being nested, the afterEach of 'Happy Path Unit Tests' seems to apply, but I will add one just to be safe.